### PR TITLE
Added support for command aliases

### PR DESCRIPTION
**Pull request by muesli**
Happy to update the README, too, if this gets accepted.

Note: the performance of this change could be slightly improved, but would require breaking the existing API. I don't expect anyone to use ishell with many thousands sub-commands though, so I preferred keeping a stable API.

### CODE REVIEW

File: command.go [hunk 1]
```diff
@@ -43,6 +45,25 @@ func (c *Cmd) DeleteCmd(name string) {
 	delete(c.children, name)
 }
 
+// CommandForName returns the subcommand with matching name or alias.
+func (c *Cmd) CommandForName(name string) *Cmd {
```
Comment (abiosoft) [inline command.go]: Let's make this a private method and rename it to something a bit better. e.g. `findChild`

**Comment by muesli** [comment]
Found findChild a bit to generic so went with `findChildCmd` to go along with its caller `FindCmd`. Let me know if that works for you.

**Comment by abiosoft** [comment]
Yeah, that works.
> Found findChild a bit to generic so went with findChildCmd to go along with its caller FindCmd. Let me know if that works for you.

**Comment by abiosoft** [comment]
One last thing, though the test coverage is very low, there are some tests in `command_test.go` and it should be straightforward to add one for `findCmd`.

Thanks.

**Comment by muesli** [comment]
There we go!

**Pull request merged**